bundles/org.openhab.binding.verisure/src/main/java/org/openhab/binding/verisure/internal/handler/VerisureBridgeHandler.java

```diff
@@ -123,15 +123,15 @@ public void initialize() {
                         if (!session.initialize(authstring, pinCode, config.username, config.password)) {
                             updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.HANDLER_REGISTERING_ERROR,
```

Comment:
HANDLER_REGISTERING_ERROR looks inappropriate. Please rather use COMMUNICATION_ERROR.

Reply:
Changed to COMMUNICATION_ERROR.